src/toolkit/date.ts

```diff
@@ -66,3 +66,30 @@ export function toHuman(
   )
   return dur2.toHuman(opts)
 }
+
+export function formatMillis(d: number): String {
```

Comment:
Try to add example, so we can distinct the different with toHuman().

Reply:
toHuman:  2hr, 51min, 22sec
formatMillis:  2h 51m 22s

Reply:
I mean in the script, so we can lookup the different directly on script later 😆

Reply:
oh, hahaha I know what you mean, okay

Reply:
done, please check